src/modules/functionsServer.ts

```diff
@@ -1,18 +1,24 @@
-import { APIGatewayEvent, Context } from 'aws-lambda'
-import * as bodyParser from 'body-parser'
+import { APIGatewayProxyEventV2, Context } from 'aws-lambda'
 import cors from 'cors'
 import express, { Application, Request, Response } from 'express'
 import { readdirSync } from 'fs'
 import path from 'path'
+import { lambdaEvent } from './utils/lambdaEvent'
 
-export const functionsServer = async (basePath: string, port: number) => {
-  const app: Application = express()
+function flattenArrays(data: Record<string, any>) {
```

Comment:
Maybe the name flattenArrays ins't the best name, since it receives an object instead of an array